packages/core/src/components/AutocompleteField/AutocompleteField.scss

```diff
@@ -0,0 +1,117 @@
+// Form-specific variables can be found in _variables.forms
+@import '@cmsgov/design-system-support/src/settings/index';
+
+/*
+Autocomplete field
+
+@paypal https://github.com/paypal/downshift#itemtostring
+
+Autocomplete fields allow users to enter any combination of letters, numbers, or symbols of their choosing (unless otherwise restricted), and receive one or more suggested matches in a list below the input.
+
+Markup:
```

Comment:
I'll rarely say this 😅  but we could remove this HTML example – I'm not sure how useful it is since there's no way to preview the autocomplete list.

Reply:
Done. Removed the HTML example, because yeah, we won't be able to show the typeahead without React.